apps/web/src/pages/templates/editor/Preview.tsx

```diff
@@ -13,6 +15,7 @@ export const Preview = ({ activeStep }: { activeStep: number }) => {
   const contentType = watch(`steps.${activeStep}.template.contentType`);
   const htmlContent = watch(`steps.${activeStep}.template.htmlContent`);
   const editorContent = watch(`steps.${activeStep}.template.content`);
+  const preheader = watch(`steps.${activeStep}.template.preheader`);
```

Comment:
In general (we have quite a few places) we need to look at the other watch options - like a callback or the useWatch hook. As this way I think it triggers unnecessary re-renders.